functions/gm.fish

```diff
@@ -0,0 +1,3 @@
+function gm
+  git commit -am "$argv"
```

Comment:
I think that input argument should be defined as `$argv` without `"` sign.

`git commit -am $argv`